src/language-js/printer-estree.js

```diff
@@ -5918,14 +5919,28 @@ function printComment(commentPath, options) {
       return "/*" + comment.value + (isInsideFlowComment ? "*-/" : "*/");
     }
     case "CommentLine":
-    case "Line":
+    case "Line": {
+      const isInJSXAttribute = commentPath.getNode(1).type === "JSXAttribute";
+
+      const isLineBeforeEmpty = isPreviousLineEmpty(
+        options.originalText,
+        comment,
+        options
+      );
+
+      const keepBlankLineAbove = isInJSXAttribute && isLineBeforeEmpty;
```

Comment:
Can you add a comment that explain why this is here? It will helps us in future.

Reply:
Done!

Reply:
Looks good to me! Let's wait for reviews of other maintainers.

Reply:
Thanks!